backend/app/api/routes/auth.py

```
@@ -0,0 +1,138 @@
+        if not profile:
+            raise HTTPException(status_code=404, detail="User profile not found.")
+
+        return LoginResponse(
+            message="Login successful.",
+            user_id=user.id,
+            email=user.email,
+            role=profile.get("role"),
+            name=profile.get("name")
+        )
+    except HTTPException as he:
+        raise he
```

Ruff (0.14.3)

Use `raise` without specifying exception name

Remove exception name

(TRY201)